tracer/src/Datadog.Trace/Ci/Test.cs

```diff
@@ -189,6 +189,154 @@ public void SetParameters(TestParameters parameters)
         }
     }
 
+    /// <summary>
+    /// Set benchmark metadata
+    /// </summary>
+    /// <param name="hostInfo">Host info</param>
+    /// <param name="jobInfo">Job info</param>
+    public void SetBenchmarkMetadata(BenchmarkHostInfo hostInfo, BenchmarkJobInfo jobInfo)
+    {
+        ((TestSpanTags)_scope.Span.Tags).Type = TestTags.TypeBenchmark;
+
+        // Host info
+
+        if (hostInfo.ProcessorName is not null)
+        {
+            SetTag(BenchmarkTestTags.HostProcessorName, hostInfo.ProcessorName);
+        }
```

Comment:
nit: could _maybe_ extract this to a helper func e.g:

```suggestion        
        SetTagIfNotNull(BenchmarkTestTags.HostProcessorName, hostInfo.ProcessorName);
        SetTagIfNotNull(BenchmarkTestTags.ProcessorCount , hostInfo.ProcessorCount );
        
        void SetTagIfNotNull(string tag, string value)
        {
            if (value is not null)
            {
                SetTag(tag, value);
            }
        }
```

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3774/commits/83258a6d8396e8ff551e7e941baccff1166d9498